src/main/config/config.c

```diff
@@ -229,6 +245,10 @@ void resetPidProfile(pidProfile_t *pidProfile)
     pidProfile->accelerationLimitPercent = 20;
     pidProfile->itermThrottleGain = 10;
 
+    for (int rI = 0; rI<MAX_RATEPROFILES; rI++) {
+        resetControlRateConfig(&masterConfig.profile[0].controlRateProfile[rI]);
+    }
+
```

Comment:
This seems to only work for profile 1 ? I think resetPidProfile() needs to be passed entire profile for this, i.e. profile_t instead of pidProfile_t